cg/models/cg_config.py

```diff
@@ -42,6 +42,11 @@ class SlurmConfig(BaseModel):
     qos: SlurmQos = SlurmQos.LOW
 
 
+class Encryption(BaseModel):
+    encryption_dir: str
+    binary_path: str
+
+
 class EncryptionDirectories(BaseModel):
     current: str
     nas: str
```

Comment:
Do they relate to each other in some way? Are both needed?

Reply:
Done